python/README.md

```diff
@@ -17,39 +17,38 @@ Then, install the dependencies:
 
     pip install -r requirements.txt
 
-```sh
```

Comment:
for `pip install -r requirements.txt`: what directory are you assuming the user is in?
Should this change to ./artiracts/requirements or should this add a cd ./artifacts

Reply:
I ended up moving the instructions for using the artifact into the artifact